.github/workflows/release.yml

```
@@ -0,0 +1,29 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      semver:
+        description: 'The semver to use'
+        required: true
+        default: 'patch'
+  pull_request:
+    types: [closed]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Optionally configure your preferred runtime
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
+      - uses: nearform/optic-release-automation-action@v2.4.0
+        with:
+          github-token: ${{ secrets.github_token }}
+          semver: ${{ github.event.inputs.semver }}
+          sync-semver-tags: true
+          build-command: |
+            npm ci
+            npm run build
```

I like adding an `echo` to make it explicit what we are doing

The installation is enough 👍🏽 

```suggestion
          build-command: |
            echo 'installing husky hooks..'
            npm ci
```

Here is an example:
https://github.com/nearform/optic-release-automation-action/pull/39